qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -217,6 +218,8 @@ def run(
 
             result_decoder: A :class:`ResultDecoder` subclass used to decode job results.
                 ``ResultDecoder`` is used if not specified.
+            runtime: The runtime image used to execute the program, specified in the form
+                of image_name:tag. Requires near-time-systems-image role.
```

Comment:
Can we use a different name, such as `image`, instead of `runtime`? `runtime` can be quite confusing since it's also the name of the service.

Reply:
Also don't mention the role since it's internal only and not something a user can see/set/ask for. Maybe just a generic statement "not all accounts are authorized to select a different image".